### ISSUE TITLE
[508 SEMANTIC] - Certificate of Eligibility review & submit page loan history html

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#46916]
[508-defect-1 :exclamation: Launchblocker](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## User Story or Problem Statement

Two axe-errors are found on the COE review & submit page when the loan history accordion is expanded:
- `<dl> elements must only directly contain properly-ordered <dt> and <dd> groups, \x3Cscript>, <template> or <div> elements` x1
- `<dt> and <dd> elements must be contained by a <dl>` x40

## Details

When using the `maximal-test.json` form data, 2 loan history blocks will appear on the review & submit page. The loan history HTML includes a `<dl class="review">` wrapping several nested `<div>`s before wrapping a `<div class="review">` which _should_ be `<dl class="review">`. The outer `<dl>` wrapper needs to be replaced with a `<div>`.

A `<dl>` can only include `<dt>`, `<dd>` or a `<div>` wrapping the `<dt>` and `<dd>` (one level deep).

## Acceptance Criteria

Example:
- [x] Properly wrap all the loan history card `<dt>` and `<dd>` inside a `<dl>` properly such that the axe-core test passes
- [x] Ensure e2e tests passing

## Environment

* Operating System: All
* Browser: All
* Screenreading device, if applicable: N/A
* Server destination (localhost, Docker container, staging, production): All except production

## Steps to Recreate

1. Go to https://staging.va.gov/housing-assistance/home-loans/request-coe-form-26-1880/introduction
2. Log into user 228 (Colder Polarbear)
3. Start the form
4. Enable the [save-in-progress menu](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-how-to-use-the-save-in-progress-m)
5. Open the save-in-progress menu
6. In a new tab, use "Copy raw contents" from the [COE maximal test data](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/lgy/coe/form/tests/fixtures/data/maximal-test.json)
7. Paste in the data into the save-in-progress menu
8. Set the return url to the review & submit page
9. Use the save-in-progress "Replace" button
10. Expand the "Your VA loan history accordion"
11. Run axe dev tools

## Proposed Solution (if known)

Discussed above

## Screenshots or Trace Logs

<img width="1267" alt="review & submit page showing html with dl wrapping 4 nested divs before reaching a div which should be a dl" src="https://user-images.githubusercontent.com/136959/189715857-58b7fc9b-8434-4b91-bcce-5494dcc2a990.png">
